tests/python/gaia-ui-tests/gaiatest/apps/gallery/regions/view_image.py

```diff
@@ -16,7 +16,8 @@ class ViewImage(Base):
 
     def __init__(self, marionette):
         Base.__init__(self, marionette)
-        Wait(self.marionette).until(lambda m: self.apps.displayed_app.name == 'Gallery')
+        from gaiatest.app.Gallery.app import Gallery
+        Wait(self.marionette).until(lambda m: self.apps.displayed_app.manifest_url == Gallery.manifest_url)
```

Comment:
As a corollary of https://github.com/mozilla-b2g/gaia/pull/31259/files#r37965728, I'd use:
```python
from gaiatest.app.Gallery.app import Gallery
Gallery.wait_to_be_displayed()
```